.github/labeler.yml

```diff
@@ -0,0 +1,273 @@
+### - integration-tests/.project
+### - integration-tests/.settings
+### - integration-tests/amazon-dynamodb
+### - integration-tests/class-transformer
+### - integration-tests/flyway
+### - integration-tests/jackson
+### - integration-tests/jgit
+### - integration-tests/jsch
+### - integration-tests/jsonb
+### - integration-tests/keycloak
+### - integration-tests/keycloak-authorization
+### - integration-tests/main
+### - integration-tests/pom.xml
+### - integration-tests/quartz
+### - integration-tests/shared-library
+### - integration-tests/test-extension
+### - integration-tests/virtual-http
+
+### Non-covered area/
+### area/bootstrap-maven
+### area/build
+### area/camel
+### area/core
+### area/dependencies
+### area/dev-mode
+### area/gradle
+### area/housekeeping
+### area/infra
+### area/new-http
+### area/oidc-interoperability
+### area/openapi
+### area/platform
+### area/reactive-messaging
+### area/security-consolidation
+### area/testing
+### area/user experience
+area/amazon-lambda:
+ - extensions/amazon-lambda/**/*
+ - extensions/amazon-lambda-http/**/*
+ - integration-tests/amazon-lambda/**/*
+ - integration-tests/amazon-lambda-http/**/*
+ - integration-tests/amazon-lambda-http-it/**/*
+ - integration-tests/amazon-lambda-http-resteasy/**/*
+
+area/arc:
+ - extensions/arc/**/*
+
+area/artemis:
+ - extensions/artemis-core/**/*
+ - extensions/artemis-jms/**/*
+ - integration-tests/artemis-core/**/*
+ - integration-tests/artemis-jms/**/*
+
+area/config:
+ - extensions/config-yaml/**/*
+
+area/documentation:
+ - docs/**/*
+
+area/hibernate-orm:
+ - extensions/hibernate-orm/**/*
+ - integration-tests/hibernate-orm-panache/**/*
+
+area/hibernate-search:
+ - extensions/hibernate-search-elasticsearch/**/*
+ - integration-tests/hibernate-search-elasticsearch/**/*
+
+area/hibernate-validator:
+ - extensions/hibernate-validator/**/*
+ - integration-tests/hibernate-validator/**/*
+
+area/infinispan:
+ - extensions/infinispan-client/**/*
+ - extensions/infinispan-embedded/**/*
+ - integration-tests/infinispan-cache-jpa/**/*
+ - integration-tests/infinispan-client/**/*
+ - integration-tests/infinispan-embedded/**/*
+
+area/jaeger:
+ - extensions/jaeger/**/*
+
+area/jaxb:
+ - extensions/jaxb/**/*
+
+area/kafka:
+ - extensions/kafka-client/**/*
+ - integration-tests/kafka/**/*
+
+area/kafka-streams:
+ - extensions/kafka-streams/**/*
+ - integration-tests/kafka-streams/**/*
+
+area/kogito:
+ - extensions/kogito/**/*
+ - integration-tests/kogito/**/*
+ - integration-tests/kogito-maven/**/*
+
+area/kotlin:
+ - extensions/kotlin/**/*
+ - integration-tests/kotlin/**/*
+
+area/kubernetes:
+ - extensions/kubernetes/**/*
+ - extensions/kubernetes-client/**/*
+ - integration-tests/kubernetes/**/*
+ - integration-tests/kubernetes-client/**/*
+
+area/logging:
+ - extensions/logging-gelf/**/*
+ - extensions/logging-json/**/*
+ - extensions/logging-sentry/**/*
+ - integration-tests/logging-gelf/**/*
+
+area/maven:
+ - integration-tests/maven/**/*
```

Comment:
Shouldn't we add the Maven plugin directory too?

Reply:
yeah - after submitting it I realized the whole `devtools` system was missing which is actually my personal interest to get auto-labelled :) I'll get that added.